extras/docker/fromsource/heketi-start.sh

```diff
@@ -12,30 +14,29 @@ if [[ -d ${HEKETI_PATH} ]]; then
     fi
 
     if [[ ! -f ${HEKETI_PATH}/heketi.db ]]; then
-        echo "" > ${HEKETI_PATH}/container.log
-        echo "DB file not found. heketi service will create a new DB" > ${HEKETI_PATH}/container.log
-        mount | grep ${HEKETI_PATH} | grep heketidbstorage >> ${HEKETI_PATH}/container.log
-        mount | grep ${HEKETI_PATH} | grep heketidbstorage
+        echo "No database file found" | tee "${HEKETI_PATH}/container.log"
```

Comment:
oh, also here, you fix the stuff that I criticized in the first commit. should be squashed. don't introduce a bad thing and fix it later in the same patchset. ;-)